azure-functions-durable/src/client.rs

```diff
@@ -93,6 +94,21 @@ pub struct Client {
     client: hyper::Client<hyper::client::HttpConnector>,
 }
 
+async fn body_from_response(res: hyper::Response<hyper::Body>) -> Result<Vec<u8>> {
```

Comment:
Was this added over using `try_concat` for efficiency?

Reply:
More for getting types to line up. try_concat takes some fun type gymnastics. Can switch to that if that is preferred.

Reply:
No preference.  Your implementation looks good to me.